libfwupdplugin/fu-udev-device.c

```diff
@@ -163,6 +174,42 @@ fu_udev_device_to_string(FuDevice *device, guint idt, GString *str)
 #endif
 }
 
+static gboolean
+fu_udev_device_ensure_bind_id(FuUdevDevice *self, GError **error)
+{
+	FuUdevDevicePrivate *priv = GET_PRIVATE(self);
+
```

Comment:
maybe a return val if fail if subsystem is null?

Reply:
I think my logic was that it would get a proper GError if NULL at the end of the function. e.g. `cannot derive bind-id from subsystem NULL`